### PR TITLE
[Reply] Contextual bottom app bar and darkmode

### PR DESCRIPTION
**Pull request by Renzo-Olivares**

### CODE REVIEW

**Comment by Renzo-Olivares** [comment]
All changes in this pull request have already been approved before hand.

File: lib/studies/reply/adaptive_nav.dart [hunk 1]
```diff
@@ -260,7 +263,7 @@ class _NavigationRailHeader extends StatelessWidget {
                         ).animate(animation),
                         child: const Icon(
                           Icons.arrow_left,
-                          color: ReplyColors.blue50,
+                          color: ReplyColors.white50,
```
Comment (guidezpl) [inline lib/studies/reply/adaptive_nav.dart:343]: double checking, these icon colors works in both light and dark theme?

File: lib/studies/reply/adaptive_nav.dart [hunk 2]
```diff
@@ -816,15 +838,31 @@ class _ReplyFab extends StatelessWidget {
...
-        child: const Icon(Icons.create),
+        tooltip: onMailView ? 'Reply' : 'Compose',
+        child: AnimatedSwitcher(
+          duration: const Duration(milliseconds: 350),
+          transitionBuilder: (child, animation) => ScaleTransition(
+            child: child,
+            scale: animation,
+          ),
+          child: onMailView
+              ? Icon(
```
Comment (guidezpl) [inline lib/studies/reply/adaptive_nav.dart]: nit: these Icons should each fit on one line. not sure that the key is necessary

File: lib/studies/reply/app.dart [hunk 3]
```diff
@@ -66,13 +79,57 @@ ThemeData _buildReplyLightTheme(BuildContext context) {
...
-    textTheme: _buildReplyTextTheme(base.textTheme),
+    textTheme: _buildReplyLightTextTheme(base.textTheme),
   );
 }
 
-TextTheme _buildReplyTextTheme(TextTheme base) {
+ThemeData _buildReplyDarkTheme(BuildContext context) {
+  final base = ThemeData.dark();
+  return base.copyWith(
+    scaffoldBackgroundColor: ReplyColors.black900,
```
Comment (guidezpl) [inline lib/studies/reply/app.dart]: nit: order these alphabetically

File: lib/studies/reply/mail_card_preview.dart [hunk 4]
```diff
@@ -147,27 +163,30 @@ class _MailPreviewActionBar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final isDark = Theme.of(context).brightness == Brightness.dark;
+    final color = isDark ? ReplyColors.white50 : ReplyColors.blue600;
```
Comment (guidezpl) [inline lib/studies/reply/mail_card_preview.dart:268]: is there a way to use theming instead?

File: lib/studies/reply/mail_view_page.dart [hunk 5]
```diff
@@ -16,20 +17,27 @@ class MailViewPage extends StatelessWidget {
...
-              ),
-              const SizedBox(height: 32),
-              Expanded(
-                child: _MailViewBody(message: email.message),
-              ),
-            ],
+        bottom: false,
+        child: Container(
+          color: Theme.of(context).cardColor,
+          child: Padding(
```
Comment (guidezpl) [inline lib/studies/reply/mail_view_page.dart]: nit: use `Container.padding` instead

**Review by guidezpl — APPROVED** [review]
```
  _    ___ _____ __  __ 
 | |  / __|_   _|  \/  |
 | |_| (_ | | | | |\/| |
 |____\___| |_| |_|  |_|
                        
```

File: lib/studies/reply/adaptive_nav.dart (hunk 1 above)
Comment (Renzo-Olivares) [reply lib/studies/reply/adaptive_nav.dart:343]: Yes this color is visible in both modes.

File: lib/studies/reply/mail_card_preview.dart (hunk 4 above)
Comment (Renzo-Olivares) [reply lib/studies/reply/mail_card_preview.dart:268]: Will address this in another PR.

**Pull request merged**